### PR TITLE
`Documentation`: Update README of Apollon

### PR DESCRIPTION
**Pull request by minrows**
### Motivation and Context
This PR simply updates the README of the Apollon project.

### Steps for Testing
1. Goto the [URL](https://github.com/ls1intum/Apollon/tree/chores/update-documentation).
2. Observe the updated readme of the project

### CODE REVIEW

**Review by mertyldrr — APPROVED** [review]
Easy to read and informative

File: README.md [hunk 1]
```diff
@@ -11,11 +11,66 @@
...
 
+## Main Features
+
+### Easy to use editor
+The user interface of Apollon is simple to use. 
+It works just like any other office and drawing tool that most users are familiar with. 
+
+-   Select the diagram type you want to draw from the `Diagram Type` menu. This selection determines the availability of elements that the user can use while drawing their diagram, making it easier for users who are newly introduced to modeling.
+-   Adding the element is as easy as dragging it from the elements menu and dropping it to the canvas. So is drawing the connection between them, simply drag and connect two or multiple elements.
+-   Edit the text of any elements by double-clicking on them. An easy-to-use menu will allow you to do so.
```
Comment (akesfeden) [inline README.md:22]:
```suggestion
-   Edit the text of any element by double-clicking on it. An easy-to-use menu will allow you to do so.
```
I think when used in a non-question form, it should be `any + singular noun`

File: README.md [hunk 2]
```diff
@@ -11,11 +11,66 @@
...
+-   Supports exporting the entire diagram or just selected elements of it.
+
+![Apollon features](/docs/images/features.gif "Apollon features")
+
+### Flexible layout features while drawing
+Apollon allows user to draw their diagram more flexibly.
+You can use keyboard shortcuts to copy, paste, delete and move the elements throughout the canvas.
+The layout of the connection is drawn automatically by the editor. 
+However, if you want to manually layout the relationship path, use the existing waypoints feature to do so.
+All you have to do is hover on the relationship, and simply drag the displayed waypoint as required.
```
Comment (akesfeden) [inline README.md:34]:
```suggestion
All you have to do is to hover over the relationship, and simply drag the displayed waypoint as required. 
```

File: README.md [hunk 3]
```diff
@@ -11,11 +11,66 @@
...
+
+### Integrate Apollon with other software
+Apollon can be integrated to any other Javascript application.
+It serves as the modeling editor for the widely used interactive learning platform called [Artemis](https://artemis.ase.in.tum.de/).  
+It also provides the standalone version of the editor.
+You can try the standalone version of Apollon completey free and without the necessity of creating any account. 
+It is a web application that allows users to use Apollon editor directly from their browser with additional features, including but not limited to, sharing and exporting the diagram.
+It can be accessed via https://apollon.ase.in.tum.de.
+
+The GitHub repository of its Standalone version is `https://github.com/ls1intum/Apollon_standalone`
```
Comment (akesfeden) [inline README.md:67]:
```suggestion
The GitHub repository of its Standalone version is https://github.com/ls1intum/Apollon_standalone
```
Just a suggestion: I think making the clickable would make it easier to navigate.

**Review by akesfeden — COMMENTED** [review]
The additions look nice, I only found some grammatical improvements and have a small suggestion.

**Review by akesfeden — APPROVED** [review]
Thanks for incorporating the requested changes, I am approving.

**Review by rriyaldhi — APPROVED** [review]
The documentation is easy to understand. Thanks for adding it!

**Pull request merged**